frame/staking/src/lib.rs

```diff
@@ -2135,6 +2249,81 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Update the various staking limits this pallet.
+		///
+		/// * `min_nominator_bond`: The minimum active bond needed to be a nominator.
+		/// * `min_validator_bond`: The minimum active bond needed to be a validator.
+		/// * `max_nominator_count`: The max number of users who can be a nominator at once.
+		///   When set to `None`, no limit is enforced.
+		/// * `max_validator_count`: The max number of users who can be a validator at once.
+		///   When set to `None`, no limit is enforced.
+		///
+		/// Origin must be Root to call this function.
+		///
+		/// NOTE: Existing nominators and validators will not be affected by this update.
+		/// to kick people under the new limits, `chill_other` should be called.
+		#[pallet::weight(T::WeightInfo::update_staking_limits())]
+		pub fn update_staking_limits(
+			origin: OriginFor<T>,
+			min_nominator_bond: BalanceOf<T>,
+			min_validator_bond: BalanceOf<T>,
+			max_nominator_count: Option<u32>,
+			max_validator_count: Option<u32>,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+			let min_nominator_bond = min_nominator_bond.max(T::Currency::minimum_balance());
+			let min_validator_bond = min_validator_bond.max(T::Currency::minimum_balance());
+			MinNominatorBond::<T>::set(min_nominator_bond);
+			MinValidatorBond::<T>::set(min_validator_bond);
+			MaxNominatorsCount::<T>::set(max_nominator_count);
+			MaxValidatorsCount::<T>::set(max_validator_count);
+			Ok(())
+		}
+
+		/// Declare a `controller` as having no desire to either validator or nominate.
+		///
+		/// Effects will be felt at the beginning of the next era.
+		///
+		/// The dispatch origin for this call must be _Signed_, but can be called by anyone.
+		///
+		/// If the caller is the same as the controller being targeted, then no further checks
+		/// are enforced. However, this call can also be made by an third party user who witnesses
+		/// that this controller does not satisfy the minimum bond requirements to be in their role.
+		///
+		/// This can be helpful if bond requirements are updated, and we need to remove old users
+		/// who do not satisfy these requirements.
+		///
+		/// TODO: Maybe we can deprecate `chill` in the future.
```

Comment:
```suggestion
		/// TODO: Maybe we can deprecate `chill` in the future.
	       // https://github.com/paritytech/substrate/issues/9111
```
Do we want the TODO in the doc comment?

Reply:
also looks like the indentation in my suggestion is wack